packages/hawtio/src/plugins/shared/chart/Chart.tsx

```diff
@@ -68,11 +78,16 @@ export const Chart: React.FunctionComponent = () => {
   }
 
   function updateNode(mbeanObjectName: string, data: MBeanChartDataEntriesTime): void {
-    if (!chartData[mbeanObjectName]) chartData[mbeanObjectName] = []
+    if (!chartData[mbeanObjectName]) chartData[mbeanObjectName] = { data: [], min: Number.MAX_SAFE_INTEGER }
 
     // Don't add repeated responses to avoid duplicate points
-    if (!chartData[mbeanObjectName]?.find(value => value.time === data.time)) {
-      chartData[mbeanObjectName]?.push(data)
+    if (!chartData[mbeanObjectName]?.data?.find(value => value.time === data.time)) {
+      chartData[mbeanObjectName]?.data?.push(data)
```

Comment:
does ts still require these `?`?  it seems redundant after we check their existence at the above `if`.

Reply:
hmm it looks I haven't commited everything. In my last iteration I don't have this. Same as  above.